ParalayoutTests/InterpolationTests.swift

```diff
@@ -29,4 +31,248 @@ final class InterpolationTests: XCTestCase {
         XCTAssertEqual(actualRect, expectedRect)
     }
 
+    // MARK: - Float
+
+    func testLinearFloatInterpolation_fromMinToMax() {
+        let value: Float = 3.0
+        let min: Float = 1.0
+        let max: Float = 5.0
+        let interpolation = Interpolation(of: value, from: min, to: max)
+            .interpolate(from: min, to: max)
+
+        XCTAssertEqual(interpolation, 3)
+    }
+
+    func testEaseInFloatInterpolation_fromMinToMax() {
+        let value: Float = 3.0
+        let min: Float = 1.0
+        let max: Float = 5.0
+        let interpolation = Interpolation(of: value, from: min, to: max)
+            .interpolate(from: min, to: max, curve: .easeIn)
+
+        XCTAssertEqual(interpolation, 2)
```

Comment:
I suspect we may need to add an `accuracy` parameter here to account for slight rounding errors across machines (for all of these assertions).

Reply:
Hm, I'm not sure what do you mean, would you mind elaborating? I haven't seen any rounding issues, the numbers in this calculation add up to `2`.

Reply:
Hey @NickEntin! I'm going to merge, but if you have any suggestions or concerns, I can address those in my next PRs!

Reply:
Sorry, forgot to respond to that one. As long as the builds are passing, I think it's fine for now. 👍